features/metadata.feature

```diff
@@ -35,7 +35,6 @@ Feature: Metadata
     Background:
         Given a metadata starts with an empty object
 
-    @ignore
     Scenario Outline: Adding some data to metadata
```

Comment:
Oh, I understood. Fix to combine `Adding some data to metadata` and `Store a record of the metadata when the event is done`. Like this?